CryptoScan.Web/client-app/src/Http.ts

```diff
@@ -0,0 +1,24 @@
+export async function get(apiEndpoint: string) {
+  return request(apiEndpoint, "GET");
+}
+
+export async function post(apiEndpoint: string, body: string | null = null) {
```

Comment:
Does it have to be nullable? I'd avoid if possible

Reply:
I tried to assign empty string to that, but it failed:
Uncaught (in promise) TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.